packages/web/src/scss/settings/_featureFlags.scss

```
@@ -0,0 +1 @@
+$enable-dropdown-enhanced-shadow: false !default;
```

I'm wondering about putting component name at start:

```suggestion
$dropdown-enable-enhanced-shadow: false !default;
```

What do you think? 🤔